pkg/util/exec/fake_exec.go

```diff
@@ -143,3 +145,69 @@ func (fake *FakeExitError) Exited() bool {
 func (fake *FakeExitError) ExitStatus() int {
 	return fake.Status
 }
+
+func commandAsStringSlice(command interface{}) []string {
+	if args, ok := command.([]string); ok {
+		return args
+	} else if commandstr, ok := command.(string); ok {
+		return strings.Split(commandstr, " ")
```

Comment:
This is not actually correct.  `foo "bar bat"` should produce `["foo", "bar bat"]`.  I don't really want to have a parsing function here - what if we just demand []string?